testsuite/features/reposync/srv_sync_products.feature

```diff
@@ -64,9 +64,9 @@ Feature: Synchronize products in the products page of the Setup Wizard
     Then I should see the "SUSE Linux Enterprise Server 15 SP4 x86_64" selected
     And I should see the "Basesystem Module 15 SP4 x86_64" selected
     # Comment following 3 lines if you wish to re-enable testing with beta client tools for SLE15
-#    And I open the sub-list of the product "Basesystem Module 15 SP4 x86_64"
-#    And I open the sub-list of the product "SUSE Manager Client Tools for SLE 15 x86_64" on SUSE Manager
-#    And I deselect "SUSE Manager Client Tools Beta for SLE 15 x86_64 (BETA)" as a SUSE Manager product
+    # And I open the sub-list of the product "Basesystem Module 15 SP4 x86_64"
+    # And I open the sub-list of the product "SUSE Manager Client Tools for SLE 15 x86_64" on SUSE Manager
+    # And I deselect "SUSE Manager Client Tools Beta for SLE 15 x86_64 (BETA)" as a SUSE Manager product
```

Comment:
But... this is Head branch, then we want to have Beta Client Tools selected.

Reply:
Correct. In this PR I just assume the code I am touching is correct.